packages/tools/pluggable-widgets-tools/configs/rollup.config.native.js

```diff
@@ -229,6 +231,23 @@ export default async args => {
     }
 };
 
+function execShellCommand(cmd, workingDirectory = process.cwd()) {
```

Comment:
Do you want to extract a common util to use? This is pretty much the same as `execShellCommand` from `createNativeModules.js` except here there's an additional `stdout` log - extracting out is an opportunity to reduce maintenance and differences between implementations.

Reply:
Yes that's also what I thought at first but this has to be part of the piw-tools while `createNativeModules.js` is not. 
The source code in piw-tools is also typescript which we can not use directly in `rollup.config.js` and `createNativeModules.js`. So I don't think we can easily do this unfortunately.